Sources/TSCUtility/Context.swift

```diff
@@ -10,8 +10,46 @@
 
 import Foundation
 
-/// Typealias for an any typed dictionary for arbitrary usage to store context.
-public typealias Context = [ObjectIdentifier: Any]
+public struct Context: /* until we can remove the support for 'Any' values */ @unchecked Sendable {
+    private var backing: [ObjectIdentifier: Any] = [:]
+
+    @available(*, deprecated, message: "Values should be Sendable")
+    @_disfavoredOverload
+    public init(dictionaryLiteral keyValuePairs: (ObjectIdentifier, Any)...) {
+        self.backing = Dictionary(uniqueKeysWithValues: keyValuePairs)
+    }
+
+    public init(dictionaryLiteral keyValuePairs: (ObjectIdentifier, Sendable)...) {
+        self.backing = Dictionary(uniqueKeysWithValues: keyValuePairs)
+    }
+
+    @available(*, deprecated, message: "Values should be Sendable")
+    @_disfavoredOverload
```

Comment:
one of my favorite tricks :)

Reply:
@jakepetroules at some point we'll need `@_overloadFavouriteness(10)` if we have need more than one level ;)